src/core/prototypes/plot_segments/axis.ts

```diff
@@ -1735,12 +1741,34 @@ export function buildAxisWidgets(
                     ),
                     dropzoneOptions
                   ),
-                  manager.clearButton({ property: axisProperty }, null, true),
-                  manager.reorderWidget(
-                    { property: axisProperty, field: "categories" },
-                    { allowReset: true }
-                  )
+                  manager.clearButton({ property: axisProperty }, null, true)
+                  // manager.reorderWidget(
```

Comment:
Remove dead, uncommented code unless there is a reason to keep it.

Reply:
Fixed, thanks!